src/ComponentTester.cpp

```diff
@@ -9,11 +9,11 @@
 #include "Logger.h"
 
 ComponentTester::ComponentTester() {
-  Logger::Log("Default ComponentTester Constructor");
+  Serial.write(14024); //14024 = Log, Info, Componet Tester
```

Comment:
Shouldn't this be `Component Tester Created` instead of `Componet (typo) Tester`? Same thing for the one below.